static/js/publisher/release/actions/gaEventTracking.js

```diff
@@ -0,0 +1,22 @@
+import { triggerEventReleaseUI } from "../../../base/ga";
```

Comment:
Can you add a `actions/gaEventTracking.test.js` file with some tests in for this please? We shouldn't have any untested actions or reducers